compiler/rustc_builtin_macros/src/cfg.rs

```diff
@@ -8,17 +8,17 @@ use rustc_ast::token;
 use rustc_ast::tokenstream::TokenStream;
 use rustc_attr as attr;
 use rustc_errors::PResult;
-use rustc_expand::base::{DummyResult, ExtCtxt, MacEager, MacResult};
+use rustc_expand::base::{DummyResult, ExpandResult, ExtCtxt, MacEager, MacroExpanderResult};
 use rustc_span::Span;
 
 pub fn expand_cfg(
     cx: &mut ExtCtxt<'_>,
     sp: Span,
     tts: TokenStream,
-) -> Box<dyn MacResult + 'static> {
+) -> MacroExpanderResult<'static> {
     let sp = cx.with_def_site_ctxt(sp);
 
-    match parse_cfg(cx, sp, tts) {
+    let res = match parse_cfg(cx, sp, tts) {
```

Comment:
```suggestion
    ExpandResult::Ready(match parse_cfg(cx, sp, tts) {
```
Style nit: can just wrap whole matches into this, without separate variables, this shouldn't change formatting.
This is relevant here and in many other places.